### ISSUE TITLE
Alexa Service Removed Garage Door

### ISSUE THREAD

**Issue by Riveloper** [NorthernMan54/homebridge-alexa#473]
Alexa has removed Garage Door support from Smart Home Routines that were previously available using this plug-in.

**Comment by NorthernMan54** [comment]
What region are you in?  For me in Canada it was still working as of about 2 hours ago.

**Comment by Riveloper** [comment]
United States. Device visible under devices but will removed from actions in routines.

**Comment by Riveloper** [comment]
Today the doors have appeared again after almost a week. Nothing on my network changed. 

But I did notice they are identifying as switches again. Which is how they were at first and work as On or Off.

At some point Alexa started identifying them correctly as Garage Doors. This is when my problem started. 

Hopefully this is useful to you. Thanks.

**Comment by NorthernMan54** [comment]
This is something within Amazon itself and not something within my control unfortunately.  I found that the implementation of Garage Doors within the Amazon Alexa cloud is a bit problematic and doesn't work properly in all regions, or work as per the documentation.

With the service, all the smarts are in the plugin itself, and the homebridge cloud service just routes the messages between the plugin and the Alexa Cloud.  So if you made no changes to the plugin, then it was Amazon.

**Comment by Riveloper** [comment]
I understand. I expect this is because of the voice code requirement for opening the doors. The service would expect something like the lock controller I had worked with before.

[https://developer.amazon.com/en-US/docs/alexa/device-apis/alexa-lockcontroller.html](https://developer.amazon.com/en-US/docs/alexa/device-apis/alexa-lockcontroller.html)

Just in case it helps, we found issue was the response time expected. 

Thanks for the help! Works great and use it all the time.
